kvdb/bolt_compact.go

```diff
@@ -228,6 +229,30 @@ func (cmd *compacter) walk(db *bbolt.DB, walkFn walkFunc) error {
 	})
 }
 
+// loggableKeyName returns a printable name of the given key.
+func loggableKeyName(key []byte) string {
```

Comment:
Should we export this so we can re-use it in the PR I linked earlier? Sounds like something that fits into the `kvdb` package.

Reply:
+1